.github/workflows/pr.yml

```diff
@@ -20,9 +27,14 @@ jobs:
           path: ~/.m2
           key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
           restore-keys: ${{ runner.os }}-m2
+      - name: Cache pacts directory
+        uses: actions/cache@136d96b4aee02b1f0de3ba493b1d47135042d9c0
```

Comment:
Have you tried using actions/cache@0865c47f36e68161719c5b124609996bb5c40129? It's newer and listed in https://github.com/alphagov/pay-webhooks/settings/actions. This will unblock https://github.com/alphagov/pay-webhooks/actions/runs/2819572945.

Reply:
Good spot, will update